include/SFML/Window/Event.hpp

```diff
@@ -366,6 +375,7 @@ class Event
     ////////////////////////////////////////////////////////////
     std::variant<Closed,
                  Resized,
+                 WindowMoved,
```

Comment:
Wonder if it should just be `Moved` given that it also isn't `WindowResized`, then again Resized doesn't stand in name conflict with say `MouseMoved`.

Any opinions on this? 🤔

Reply:
If I had to guess, `Resized` isn't `WindowResized` because it didn't require the disambiguation, whereas `Moved` does.

Reply:
I chose this name because it is more explicit about what is being moved.
Imo resized should be renamed to WindowResized, but that would probably also require a typedef for backwards compatibility